prisma/seeds/teams.ts

```diff
@@ -1,15 +1,28 @@
-import type { PrismaClient, Sport } from '@prisma/client'
-
-const seedTeams = (prisma: PrismaClient, sports: Sport[]) => {
-  const rugbyId = sports.find(sport => sport.key === 'rugby')!.id
-
-  const teams = [
-    {
-      key: 'wallabies',
-      sportId: rugbyId,
-      title: 'Wallabies',
-    },
-  ]
+import { competitionTeams } from './teams/rugby'
+
+import type { Competition, PrismaClient } from '@prisma/client'
+
+const getCompetitionTeamsArray = (competitionKey: keyof typeof competitionTeams) => {
+  return competitionTeams[competitionKey] || []
+}
+
+const seedTeams = (prisma: PrismaClient, competitions: Competition[]) => {
+  const teams = competitions.flatMap(competition => {
```

Comment:
had a long think about this and i think it is indeed a good solution

Reply:
FlatEarth Baby!